src/univariates.jl

```diff
@@ -341,7 +341,6 @@ Relying on this fallback is not recommended in general, as it is prone to overfl
 """
 logpdf(d::UnivariateDistribution, x::Real) = log(pdf(d, x))
 logpdf(d::DiscreteUnivariateDistribution, x::Integer) = log(pdf(d, x))
-logpdf(d::DiscreteUnivariateDistribution, x::Real) = isinteger(x) ? logpdf(d, round(Int, x)) : partype(d)(-Inf)
```

Comment:
Was this method never called or was it never necessary?

Reply:
It's sufficient to have the conversion logic to `Int` in the `pdf` version which will then be called by the version in line 342.